src/App.test.jsx

```diff
@@ -9,44 +9,32 @@ import { useDispatch, useSelector } from 'react-redux';
 
 import App from './App';
 
+import regions from '../fixtures/regions';
```

Comment:
fixture 는 대문자로 작성하시면 어떨까요?
```suggestion
import REGIONS from '../fixtures/regions';
```

Reply:
네 fixture 사용 가능한 테스트에서는 가급적 fixture 사용하도록 수정하였고요, fixture는 대문자로 constant임을 표기하였습니다.